### ISSUE TITLE
Is there a mistake in the instruction of PU learning?

### ISSUE THREAD

**Issue by Hengwei-Zhao96** [cleanlab/cleanlab#64]
Hi, thanks for your good job, however, there is something in the instructions of PU learning that puzzles me:

Method 1. If you are using the cleanlab classifier LearningWithNoisyLabels(), and your dataset has exactly two classes (positive = 1, and negative = 0), PU learning is supported directly in cleanlab. You can perform PU learning like this:

from cleanlab.classification import LearningWithNoisyLabels
from sklearn.linear_model import LogisticRegression

**pu_class = 0** # Should be 0 or 1. Label of class with NO ERRORS. (e.g., P class in PU)

lnl = LearningWithNoisyLabels(clf=LogisticRegression(), pulearning=pu_class)
lnl.fit(X=X_train_data, s=train_noisy_labels)

predicted_test_labels = lnl.predict(X_test)

Whether pu_class=1 because of the positive class have no label noisy?

**Comment by jwmueller** [comment]
Sorry nobody got to your question before. I believe the newer version of cleanlab has clearer documentation regarding this so please check that out. I'll close the issue due to lack of activity for now, but please reopen it if you still have questions!  

If you do still have questions, it would be helpful to write a bit more detail about what exactly you think is wrong in our instructions, or more information about your source of confusion.
